CONTRIBUTING.md

```diff
@@ -11,6 +11,11 @@ Instead, modify the sources in the ```www``` folder and compile a new version of
 
 ```node.js``` and ```npm``` are needed for building this project. ```npm``` comes bundled with the ```node.js``` installer. You can download the ```node.js``` installer here: https://nodejs.org/download/. 
 
+To run Android tests, make sure you have ```sdk\tools``` and  ```sdk\platform-tools``` in your PATH.
+
+If you would like the tests to start the emulators for you, configure an Android emulator using the AVD and name it ```emulator```.
```

Comment:
Although supported, gulp command line parameters are a bit odd to use. If this is the first scenario where we need them, you could add the parameters to `mocha`, not `gulp` instead. That, coupled with moving the emulator managing code from the gulp file to the tests makes more sense.